common/views/components/MainMedia/MainMedia.js

```diff
@@ -33,11 +33,7 @@ export const MainMedia = ({
   return (
     <figure className={`captioned-image ${extraClasses}`}>
       <div className='captioned-image__image-container'>
-        {/* https://github.com/facebook/flow/issues/2405 */}
```

Comment:
Ah – was failing for me locally; I should update my Flow.

Reply:
You can use the one in `node_modules` to ensure consistency.